doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/autoscaling.md

```diff
@@ -92,3 +97,22 @@ One method is to schedule fewer tasks/actors per node by increasing the resource
 requirements specified in the `ray.remote` annotation.
 For example, changing `@ray.remote(num_cpus=2)` to `@ray.remote(num_cpus=4)`
 will halve the quantity of that task or actor that can fit in a given Ray pod.
+
+## Autoscaling architecture
+The following diagram illustrates the integration of the Ray Autoscaler
+with the KubeRay operator.
+```{eval-rst}
+.. image:: /cluster/cluster_under_construction/ray-clusters-on-kubernetes/images/AutoscalerOperator.png
```

Comment:
Very nice diagram! Maybe @scottsun94 can put it into a style that is consistent with the other graphics in the docs?

And let's also try to link the raw google docs graphs like for the other sections.

Reply:
@DmitriGekhtman Could you add a link to the diagram? I could try to make it consistent with others

Reply:
Yeah, let me follow up offline.

Reply:
I've updated it to use @scottsun94's cleaned up version.

Reply:
Noting in passing that sphinx can render svg images, which saves having to think about resolution.